craco.config.js

```diff
@@ -1,8 +1,34 @@
+const webpack = require("webpack");
+
 module.exports = {
     babel: {
         plugins: [
+            "@babel/plugin-transform-async-to-generator",
             "@babel/plugin-proposal-nullish-coalescing-operator",
             "@babel/plugin-proposal-optional-chaining",
         ],
+    },
+    webpack: {
+        configure: {
+            module: {
+                exprContextCritical: false, // turns off Critical dependency: the request of a dependency is an expression error
+            },
+            experiments: {
+                topLevelAwait: true,
+            },
+            resolve: {
+                fallback: {
+                    "crypto": require.resolve("crypto-browserify"),
+                    "http": require.resolve("stream-http"),
+                    "https": require.resolve("https-browserify"),
+                    "stream": require.resolve("stream-browserify"),
+                    "url": require.resolve("url/"),
+                    "os": require.resolve("os-browserify/browser"),
+                    "path": require.resolve("path-browserify"),
+                    "zlib": require.resolve("browserify-zlib"),
```

Comment:
Why do you need all these dependencies? yarn.lock was bumped by 11_000 lines that's insane

Reply:
Webpack 5 has major breaking changes. See [here](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed) One of which is dropping of polyfills, which totally breaks the build in many ways. 

These changes and others are to get the build running successful first, and then get the application running. After which any erroneous dependencies can be removed. 

Note this PR is still in draft and thats because it is not completed yet.